README.md

```diff
@@ -196,3 +196,13 @@ services:
 		factory: PHPStan\Type\Doctrine\Descriptors\ReflectionDescriptor('MyApp\MyCustomTypeName')
 		tags: [phpstan.doctrine.typeDescriptor]
 ```
+
+### Ensure types have descriptor
+
+If you want to be sure you never forget descriptor for some custom type, you can enable:
```

Comment:
Please be a little bit more specific here. Show a code example and what error gets reported in that example if this is enabled.